indra/llwindow/llgamecontrol.cpp

```diff
@@ -1398,8 +1399,8 @@ void onControllerDeviceAdded(const SDL_Event& event)
 {
     std::string guid(std::to_string(SDL_JoystickGetDeviceGUID(event.cdevice.which)));
     SDL_GameControllerType type(SDL_GameControllerTypeForIndex(event.cdevice.which));
-    std::string name(SDL_GameControllerNameForIndex(event.cdevice.which));
-    std::string path(SDL_GameControllerPathForIndex(event.cdevice.which));
+    std::string name(std::to_string(SDL_GameControllerNameForIndex(event.cdevice.which)));
+    std::string path(std::to_string(SDL_GameControllerPathForIndex(event.cdevice.which)));
```

Comment:
I'm not seeing an version of `std::to_string()` in the documentation that accepts a `const char*` as an argument.  Did we provide a custom override that handles the `nullptr` or... how does this work?

Reply:
Yes